c2rust-analyze/src/util.rs

```diff
@@ -89,8 +89,21 @@ pub enum Callee<'tcx> {
         mutbl: Mutability,
     },
 
-    /// A built-in or standard library function that requires no special handling.
-    MiscBuiltin,
+    /// A [`Trivial`] library function that requires no special handling.
```

Comment:
```suggestion
    /// A [`Trivial`] library function is one that has no effect on pointer permissions in its caller.
```

Since the current implementation (looking for pointers in argument/return types) only approximates the actual `Trivial` property, I think the precise definition should be included here in the comment.

Reply:
That makes sense, I agree.

Reply:
In fd48ed87242a603bced61135a612d57c182b9531, I made this change but kept the "requires no special handling" as

```rust
/// Thus, a [`Trivial`] function call requires no special handling.
```